src/webgpu/api/validation/encoding/programmable/pipeline_bind_group_compat.spec.ts

```diff
@@ -88,7 +91,36 @@ class F extends ValidationTest {
 
 export const g = makeTestGroup(F);
 
-g.test('it_is_invalid_to_draw_in_a_render_pass_with_missing_bind_groups')
+g.test('set_pipeline_or_bind_group_missing_in_render_pass')
+  .desc('Tests the bind groups required by the pipeline layout are required.')
+  .paramsSubcasesOnly(u =>
+    u
+      .combine('cmd', kDrawCmds)
+      .combine('setPipeline', [true, false] as const)
+      .combine('setBindGroup', [true, false] as const)
```

Comment:
Perhaps test two different bind groups missing, e.g., with a pipeline that requires bind groups 0 and 2:
- set bind groups [0, 2]
- set bind groups [0]
- set bind groups [2]

Reply:
Covered in `bind_groups_and_pipeline_layout_mismatch` after combination.